src/Arcus.Scripting.ApiManagement/Arcus.Scripting.ApiManagement.psm1

```diff
@@ -173,3 +173,37 @@ function Import-AzApiManagementOperationPolicy {
 
     . $PSScriptRoot\Scripts\Import-AzApiManagementOperationPolicy.ps1 -ResourceGroup $ResourceGroup -ServiceName $ServiceName -ApiId $ApiId -OperationId $OperationId -PolicyFilePath $PolicyFilePath
 }
+
+<#
+ .Synopsis
+  Sets the authentication keys in Azure API Management.
+
+ .Description
+  Sets the authentication header/query parameter on an API in Azure API Management.
+
+ .Parameter ResourceGroup
```

Comment:
```suggestion
 .Parameter ResourceGroupName
```

Reply:
We should rename them all togheter, see #95 , resolving this.